app/src/androidTest/java/ch/epfl/sdp/drone3d/ui/mission/ItineraryShowActivityTest.kt

```diff
@@ -75,19 +85,29 @@ class ItineraryShowActivityTest {
 
         activityRule.scenario.recreate()
 
-        onView(ViewMatchers.withId(R.id.buttonToMissionInProgressActivity))
-            .check(matches(ViewMatchers.isEnabled()))
-        onView(ViewMatchers.withId(R.id.buttonToMissionInProgressActivity)).perform(ViewActions.click())
+        onView(withId(R.id.buttonToMissionInProgressActivity))
+            .check(matches(isEnabled()))
+        onView(withId(R.id.buttonToMissionInProgressActivity)).perform(click())
 
         Intents.intended(
-            IntentMatchers.hasComponent(
-                ComponentNameMatchers.hasClassName(
-                    MissionInProgressActivity::class.java.name
-                )
-            )
+            hasComponent(hasClassName(MissionInProgressActivity::class.java.name))
         )
 
         val intents = Intents.getIntents()
         assert(intents.any{it.hasExtra(MissionViewAdapter.MISSION_PATH)})
     }
+
+    @Test
+    fun deleteMissionBringBackToMissionSelection() {
+        val user = Mockito.mock(FirebaseUser::class.java)
```

Comment:
We decided that we would remove Mockito here

Reply:
I had to add it to avoid a NullPointerException when returning to MissionSelectionActivity